lib/recommended-rules.js

```diff
@@ -68,6 +68,7 @@ module.exports = {
   "ember/require-tagless-components": "error",
   "ember/require-valid-css-selector-in-test-helpers": "error",
   "ember/routes-segments-snake-case": "error",
+  "ember/template-vars": "error",
```

Comment:
Note that we'll need to wait until the next major release (https://github.com/ember-cli/eslint-plugin-ember/issues/1169) before enabling this as recommended.

Reply:
Yeah, I had a hard time with the generated tests. Should i make a new config for Polaris and put this there?

Reply:
I think I prefer not to deal with managing another config for now. Why did it cause a problem with the tests? A rule doesn't need to be in a config to be tested.

Reply:
ah I don't remember things seem fine now. :upside_down_face: